### ISSUE TITLE
"Object doesn't support property or method 'undefined'" error in IE11

### ISSUE THREAD

**Issue by fredrik-sogaard** [erikras/redux-form#2700]
### Are you submitting a **bug report** or a **feature request**?
Bug

### What is the current behavior?
I use a simple Field component in my form:
```
<Field classes='mt2 w-50 pr2' name='firstName' component='input' type='text' placeholder='Add your first name' label='First name' validate={[ required ]} />
```
With a simple validator:
```
const required = value => value ? undefined : 'Required'
```

When I test in IE11 I get error `Object doesn't support property or method 'undefined'`. Other browsers are ok. If I remove the `validate={[ required ]}` prop, everything works fine.

### What is the expected behavior?
Validating should work in IE11?

### What's your environment?
Redux-from 6.5.0
IE11

### Other informations
Stack Trace:
![image 2017-03-16 at 3 26 40 pm](https://cloud.githubusercontent.com/assets/764318/24000937/ff3d77e2-0a5c-11e7-94f7-12196c3fc675.png)

Demo:
![screen recording 2017-03-16 at 03 29 pm](https://cloud.githubusercontent.com/assets/764318/24001059/5b744748-0a5d-11e7-80a5-9b3fca0cc161.gif)

**Comment by erikras** [comment]
Can you isolate which line is causing the error? i.e. what is on line 58449 of `bundle.js`?

**Comment by fredrik-sogaard** [comment]
This is the part of bundle.js that fails. Line 58449 is the `throw _iteratorError;` line. I'm using the latest Create React App to build and serve the app.

```
/***/ },
/* 566 */
/*!***********************************************!*\
  !*** ./~/redux-form/lib/generateValidator.js ***!
  \***********************************************/
/***/ function(module, exports, __webpack_require__) {

	'use strict';

	Object.defineProperty(exports, "__esModule", {
	  value: true
	});

	var _plain = __webpack_require__(/*! ./structure/plain */ 558);

	var _plain2 = _interopRequireDefault(_plain);

	function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

	var toArray = function toArray(value) {
	  return Array.isArray(value) ? value : [value];
	};

	var getError = function getError(value, values, props, validators) {
	  var _iteratorNormalCompletion = true;
	  var _didIteratorError = false;
	  var _iteratorError = undefined;

	  try {
	    for (var _iterator = toArray(validators)[Symbol.iterator](), _step; !(_iteratorNormalCompletion = (_step = _iterator.next()).done); _iteratorNormalCompletion = true) {
	      var validator = _step.value;

	      var error = validator(value, values, props);
	      if (error) {
	        return error;
	      }
	    }
	  } catch (err) {
	    _didIteratorError = true;
	    _iteratorError = err;
	  } finally {
	    try {
	      if (!_iteratorNormalCompletion && _iterator.return) {
	        _iterator.return();
	      }
	    } finally {
	      if (_didIteratorError) {
	        throw _iteratorError;
	      }
	    }
	  }
	};
```

**Comment by fredrik-sogaard** [comment]
@erikras Have you had a change to look at this?

**Comment by jerrysu** [comment]
This is breaking forms with validators for us as well in IE11. We found that the issue is with the following `for-of` loop and how babel transforms this to use `Symbol.iterator()` (https://github.com/babel/babel/issues/1534):

![screen shot 2017-03-22 at 6 43 17 pm](https://cloud.githubusercontent.com/assets/110441/24228274/6862cb14-0f30-11e7-99c3-6bb69c334207.png)


So, it seems you can fix this by including a Symbol polyfill, but we would prefer if this library can use widely-supported and more conventional means of iterating arrays. I can try to submit a PR.

**Comment by jptconiconde** [comment]
We are also experiencing this issue, I'll also try what  I can do on our end.

**Comment by jptconiconde** [comment]
I've added the runtime and it works, please verify the [PR](https://github.com/erikras/redux-form/pull/2720/commits/218e2211a2f76a48c3c81a43a6739edb6facf56d)

**Comment by fredrik-sogaard** [comment]
I have added a Symbol polyfill via https://polyfill.io, and that gets rid of the `'Symbol' is undefined` error, but I still get the `Object doesn't support property or method 'undefined'` error. It might be related?

**Comment by jerrysu** [comment]
@fredrik-sogaard: Try [6.6.0](https://github.com/erikras/redux-form/releases/tag/v6.6.0) as it addresses the issues that I had with IE11.

**Comment by gustavohenke** [comment]
@jerrysu is right. v6.6.0 fixes this - check corresponding PR: #2724
